truckersmp-cli

```diff
@@ -529,6 +536,32 @@ Please report an issue: {}""".format(e, URL.issueurl))
             sys.exit("Failed to download mod files.")
 
 
+def get_current_steam_user():
+    """
+    Get the current AccountName with saved login credentials.
+
+    If successful this returns the AccountName of the user with saved credentials.
+    Otherwise this returns None.
+    """
+    if not vdf_is_available:
+        return None
```

Comment:
Which do you think is better?

A. Only add dependency-related description to docstring

Example:
```python
    """
    ...
    Otherwise this returns None.
    If package "vdf" is not available, this returns None
    because this function depends on it.
    """
```

B. Don't change the docstring (or add something such as "This function depends on `vdf` package."), remove the `vdf_is_available` check from this function, and add the check to `check_args_errors()`

Example:
```python
     # checks for updating
     if args.update and not args.account:
         sys.exit("Need the steam account name (-n name) to update.")
         # get_current_steam_user() depends on "vdf" package
         if vdf_is_available:
             args.account = get_current_steam_user()
         if not args.account:
             logging.info("Unable to find logged in steam user automatically.")
             sys.exit("Need the steam account name (-n name) to update.")
```

I prefer "B."

Reply:
I've also thought about this before but decided to go somehow with A. But B looks better anyway :)